x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/index.ts

```diff
@@ -6,18 +6,36 @@
  */
 
 import {
-  EuiMarkdownEditorUiPlugin,
+  EuiLinkAnchorProps,
   getDefaultEuiMarkdownParsingPlugins,
   getDefaultEuiMarkdownProcessingPlugins,
   getDefaultEuiMarkdownUiPlugins,
 } from '@elastic/eui';
-
+// Remove after this issue is resolved: https://github.com/elastic/eui/issues/4688
```

Comment:
The issue seems to be resolved.

Reply:
its not merged into kibana yet i dont think

Reply:
jk it is there now, ill remove it

Reply:
jk there was a different error: 
<img width="714" alt="Screen Shot 2021-04-26 at 8 42 02 AM" src="https://user-images.githubusercontent.com/6935300/116101893-76d03480-a66b-11eb-8642-c5f103efb4d6.png">

Reply:
@chandlerprall says:
> we'll cut a release tomorrow with that change, and depending on Greg's availability we'll get a Kibana upgrade started this week or next

Reply:
I see. No worries!